security/try-pod-to-pod-encryption.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Try out the pod-to-pod encryption tech preview
+description: Try out enabling WireGuard for state-of-the-art cryptographic security between pods for Calico clusters.
+---
+
+### Big picture
+
+Enable WireGuard to secure pod-to-pod traffic in a {{site.prodname}} cluster.
+
+> **Warning!** Pod-to-pod encryption is a tech preview and should not be used in production clusters. It has had very limited testing and it will contain bugs (please report these on the Calico Users slack or Github).  In addition, it does not support all the features of {{site.prodname}}.
+{: .alert .alert-danger }
+
+### Value
+
+{{ site.prodname }} supports WireGuard tunnels between nodes providing transport-level security for pod-to-pod traffic. WireGuard provides [formally verified](https://www.wireguard.com/formal-verification/) secure and [performant tunnels](https://www.wireguard.com/performance/) without any specialized hardware. For a deep dive in to WireGuard implementation, see [whitepaper](https://www.wireguard.com/papers/wireguard.pdf).
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- [Felix resource]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition) with WireGuard configuration parameters.
+
+### Before you begin...
+
+- Verify the operating system(s) running on the nodes in the cluster [support WireGuard](https://www.wireguard.com/install/).
```

Comment:
We should also have a note equivalent to the one in the the eBPF doc:

> An underlying network that doesn’t require Calico to use an overlay. The instructions below guide you through setting up a cluster in a single AWS subnet; an alternative would be to set up your cluster on-prem with a routed network topology.

Reply:
I am adding a line in the warning section at the top which talks about issues with overlay networks. I think that should suffice. Let me know if you disagree.

Reply:
I think it belongs here - this is where we've put the other pre-requisites and limitations in comparable docs.

Reply:
OK, I will repeat it here as well then.